src/models/CreateAndRunHelper.ts

```diff
@@ -0,0 +1,326 @@
+        filesToDelete.push(testFiles.userPropFileInfo.fileName);
+    }
+
+    if(!isNullOrUndefined(testFiles.additionalFileInfo)){
+        // delete existing files which are not present in yaml, the files which are in yaml will anyway be uploaded again.
+        let file : any;
+        for(file of testFiles.additionalFileInfo){
+            filesToDelete.push(file.fileName);
+        }
+        for(let file of yamlModel.configurationFiles){
+            file = Util.getFileName(file);
+            let indexOfFile = filesToDelete.indexOf(file);
+            if(indexOfFile != -1){
+                filesToDelete.splice(indexOfFile, 1);
+            }
+        }
+        for(let file of yamlModel.zipArtifacts){
+            file = Util.getFileName(file);
+            let indexOfFile = filesToDelete.indexOf(file);
+            if(indexOfFile != -1){
+                filesToDelete.splice(indexOfFile, 1);
+            }
+        }
+        
+    }
+    return filesToDelete;
+}
+
+export function ValidateAndGetRunTimeParamsForTestRun(testId: string) {
+    var secretRun = CoreUtils.getInput(InputConstants.secrets);
+    let secretsParsed : {[key: string] : SecretMetadata} = {};
+    let envParsed : {[key: string] : string} = {};
+    if(secretRun) {
+        try {
+            var obj = JSON.parse(secretRun);
+            for (var index in obj) {
+                var val = obj[index];
+                let str : string =  `name : ${val.name}, value : ${val.value}`;
+                if(isNullOrUndefined(val.name)){
+                    throw new Error(`Invalid secret name at pipeline parameters at ${str}`);
+                }
+                secretsParsed[val.name] = {type: 'SECRET_VALUE',value: val.value};
+            }
+        }
+        catch (error) {
+            console.log(error);
+            throw new Error(`Invalid format of ${InputConstants.secretsLabel} in the pipeline file. Refer to the pipeline syntax at : https://learn.microsoft.com/en-us/azure/load-testing/how-to-configure-load-test-cicd?tabs=pipelines#update-the-azure-pipelines-workflow`);
+        }
+    }
+    var eRun = CoreUtils.getInput(InputConstants.envVars);
+    if(eRun) {
+        try {
+            var obj = JSON.parse(eRun);
+            for (var index in obj) {
+                var val = obj[index];
+                let str : string =  `name : ${val.name}, value : ${val.value}`;
+                if(isNullOrUndefined(val.name)){
+                    throw new Error(`Invalid environment name at pipeline parameters at ${str}`);
+                }
+                envParsed[val.name] = val.value;
+            }
+        }
+        catch (error) {
+            console.log(error);
+            throw new Error(`Invalid format of ${InputConstants.envVarsLabel} in the pipeline file. Refer to the pipeline syntax at : https://learn.microsoft.com/en-us/azure/load-testing/how-to-configure-load-test-cicd?tabs=pipelines#update-the-azure-pipelines-workflow`); 
```

Comment:
Same here, lets verify all the links and/or define in constants

Reply:
for now not keeping in any constants. lets take it up later.